### PR TITLE
Deployment for the version 1

### PR DESCRIPTION
**Pull request by DavidDaoTu**
The first release has been deployed on the production server

### CODE REVIEW

**Comment by DavidDaoTu** [comment]
[Deploy the first release v0.1](https://trello.com/c/ny3rG1wx/18-deploy-the-first-release-v01)

File: deployment/backend_deployment.yaml [hunk 1]
```diff
@@ -20,8 +20,8 @@ spec:
     spec:
       containers:
         - name: hgtp-backend
-          image: hgtp_backend_img:latest
-          imagePullPolicy: IfNotPresent
+          image: davidfullstack/hgtp_backend_img:latest
```
Comment (DavidDaoTu) [inline deployment/backend_deployment.yaml:23]: Nên đổi tag name thành v1

**Comment by DavidDaoTu** [comment]
Không nên để tag name của Docker image là latest khi deploy

**Review by minhduc-mll — APPROVED** [review]

**Comment by phunghaduong99** [comment]
Ok

**Review by phunghaduong99 — APPROVED** [review]

**Pull request merged**